resources/docker/Dockerfile.cgo

```diff
@@ -16,8 +23,6 @@ LABEL maintainer="Ashley Jeffs <ash@jeffail.uk>"
 
 WORKDIR /root/
 
-RUN apt-get update && apt-get install -y --no-install-recommends libzmq3-dev
```

Comment:
(Unrelated cleanup)

Reply:
I could be wrong, but since this container builds Benthos with `make TAGS=ZMQ4`, then `libzmq.so` and its dependencies need to be installed in the final container.

Reply:
Hmm, I have to say I am not sure about this, I will return it as its a no-brainer solution.
The reason I removed it is I thought it was useful for the build phase only, not sure now.